### PR TITLE
ENT-3465 Extra select fields were added to Inventory API documentation.

### PR DESCRIPTION
**Pull request by aleksandrychev**

### CODE REVIEW

File: reference/enterprise-api-ref/inventory.markdown [hunk 1]
```diff
@@ -18,7 +18,7 @@ Inventory API allows to access inventory reports and attributes dictionary.
 
 * **select** *(array)*
     Fields for selecting. Required parameter.
-    List of fields name you can obtain through [List of inventory traits][Inventory API#List of inventory traits] call described below.
+    List of fields name you can obtain through [List of inventory traits][Inventory API#List of inventory traits] call described below. Extra field names are `hostkey` - to get host key, and `resultCount` - to get count of rows.
```
[the remark this answers is not in the document]
Comment (aleksandrychev) [reply reference/enterprise-api-ref/inventory.markdown]: In select field you put variables name which you want to get in result. But if you want to get only count of rows (emp. How many hosts listening 80 port) without data you can use extra select value like resultCount. Or hostkey doesn't present in inventory attributes, but you want ti have host key ib result, you need paste into select array histkey value.

[the remark this answers is not in the document]
Comment (nickanderson) [reply reference/enterprise-api-ref/inventory.markdown]: So perhaps modify it like this:

I think we call these "attributes" not "traits".


```
* **select** *(array)*
    Inventory attributes to return. Required parameter.
    List of fields name you can obtain through [List of inventory attributes][Inventory API#List of inventory attributes] call described below. In addition to available inventory attributes, selecting  ```hostkey``` will return the hostkey identifer, and ```resultCount``` will return the number of hosts having the same value.
```

Does that sound correct?

@craigcomstock you have been nitpicking docs recently, do you have any suggestion to make this more clear?

[the remark this answers is not in the document]
Comment (craigcomstock) [reply reference/enterprise-api-ref/inventory.markdown]: "List of fields by name that you can obtain through {List of inventory attribute names][Inventory API#List of inventory attribute names] call described below.

s/selecting/including/g for selecting ```hostkey``` and selecting ```resultCount```

Are we using the term "inventory trait" or "inventory attribute" elsewhere? would be good to be consistent.

[the remark this answers is not in the document]
Comment (nickanderson) [reply reference/enterprise-api-ref/inventory.markdown]: Inventory attribute is the term we should use.

**Pull request merged**